### ISSUE TITLE
Can't find the proper .NET Core Version

### ISSUE THREAD

**Issue by lucaslra** [DXBrazil/Arda#2]
Hi,
I am trying to do a first go in the project, but I am facing a problem to open it in Visual Studio Community 2015 in Windows 10.

The problem is pretty straightforward, but I can't find a solution. The error presented just after I open the .sln file is
![image](https://cloud.githubusercontent.com/assets/2528972/20079770/64c0f066-a52d-11e6-9a6c-2ab5c9b1ddb4.png)

Problem is, I could only find the ".NET Core SDK version 1.0.0-preview2.1-003155" for Linux machines...

The obvious solution would be to change the SDK version, but then I would not be compliant with the current version you guys are using.

Am I doing something wrong?

**Comment by lucaslra** [comment]
One more piece of information, my SDK Version

![image](https://cloud.githubusercontent.com/assets/2528972/20081875/5f54b7e6-a539-11e6-84e2-2237830c1292.png)

**Comment by ViniciusSouza** [comment]
Hi Lucas, good to know that the project is alive! 👍 

I had some issues to connect to Azure DB in the current version dotnet core, so I had to upgrade to a preview one.  .NET Core 1.1 Preview 1 and .NET Core SDK 1.0 Preview 2.1 

At this link you will find the links to download https://github.com/dotnet/core/blob/master/release-notes/preview-download.md.

I got the Binaries (zip files) and I have expanded it to my donet folder.
